pkg/nfd-master/nfd-master.go

```diff
@@ -185,6 +185,15 @@ func (m *nfdMaster) Run() error {
 	klog.Infof("NodeName: %q", m.nodeName)
 	klog.Infof("Kubernetes namespace: %q", m.namespace)
 
+	// Create watcher for config file and read initial configuration
+	configWatch, err := utils.CreateFsWatcher(time.Second, m.configFilePath)
```

Comment:
do we need to setup `configWatch` for prune to work?

Reply:
Good point @PiotrProkop, we don't need it as `-prune` is sub-command like option which just cleans up the nodes and exits right away. Wouldn't do much harm to set it but sort of "beauty defect" -> fixed it